themes/default/content/docs/intro/concepts/state.md

```diff
@@ -80,7 +80,7 @@ For details on the various backend URL formats and options, please see the follo
 - [Pulumi Self-Hosted Service](#logging-into-a-self-hosted-pulumi-service-backend)
 - [Local Filesystem](#logging-into-the-local-filesystem-backend)
 - [AWS S3 (or compatible server)](#logging-into-the-aws-s3-backend)
-- [Azure Blob Storage](#logging-into-the-azure-blob-storage-backend)
+- [Azure Blob Storage](#logging-into-the-azure-blob-storage-backend-using-the-env-vars)
```

Comment:
Style: this heading should match the others, the S3 heading is an example of this

```suggestion
- [Azure Blob Storage](#logging-into-the-azure-blob-storage-backend)
```